packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -39,6 +39,45 @@ type UpdateOptions = {
   input: TUpdateInputSchema;
 };
 
+const verifyEmailSender = async (
+  email: string,
+  userId: number,
+  teamId: number | null,
+  prisma: PrismaClient
+) => {
+  const verifiedEmail = await prisma.verifiedEmail.findFirst({
+    where: {
+      email,
+      OR: [{ userId }, { teamId }],
+    },
+  });
+
+  if (!verifiedEmail) {
+    throw new TRPCError({ code: "NOT_FOUND", message: "Email not verified" });
```

Comment:
Let's show a toast with this error message when this error thrown

Reply:
https://github.com/calcom/cal.com/blob/56f31eee7ab517c844b6ad5928dfebb7d438626e/packages/features/ee/workflows/pages/workflow.tsx#L198-L204

Should be handled here